### DISCUSSION TITLE
Sunbird Obsrv local installation

### DISCUSSION THREAD

**Discussion by Kovendan03** [Sunbird-Obsrv/Community#40] [Installation]
Hi Team,

I am trying to install Sunbird Obsrv locally by following the documetation - https://obsrv.sunbird.org/use/installation-guide, docker images and manifest files are not available. Installation got failed.

**Comment by sagivpn** [comment]
Team kindly help @Kovendan03 with the query. @gandham-santhosh

**Reply by anandp504** [reply]
@Kovendan03 @sagivpn You can use the one-click install scripts in the [sunbird-infra-provision](https://github.com/Sunbird-Obsrv/sunbird-infra-provision/tree/release-5.1.0) repository to run everything either locally or on Azure on Kubernetes.

**Reply by mathew-alex** [reply]
@anandp504 - I am also trying to install Sunbird, but images and manifest files are unavailable according to the installation guides. I have also tried the one-click install scripts. but that also failed when I do not create an aks.yaml file 
Please provide me with a sample ingestion spec file (ask.yaml), that I may use.

**Reply by gigincg** [reply]
@sagivpn @anandp504 The link to obsrv-automation is invalid? Could you please help?
